fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -1107,6 +1148,8 @@ ACTOR Future<Void> commitBatch(ProxyCommitData* self,
 
 	/////// Phase 2: Resolution (waiting on the network; pipelined)
 	state vector<ResolveTransactionBatchReply> resolution = wait(getAll(replies));
```

Comment:
Can we differentiate the reply latency from each resolver?

Reply:
For example, the stats.resolutionDist can be a vector. Each element is the latency from a resolver.

When we initiate the vector, we trace which element is for which resolver.